civis/resources/_resources.py

```diff
@@ -145,7 +146,7 @@ def iterable_method(method, params):
     return (method.lower() == 'get' and params_present)
 
 
-def create_signature(args, kwargs):
+def create_signature(args, optional_args, kwargs=False):
```

Comment:
do we ever use `kwargs` here?

Reply:
No. Shall I remove it?

Reply:
Sure. We can add it back if we need it later.